src/index.js

```diff
@@ -1 +1,40 @@
 import './style.css';
+import addScore from './modules/Addscores.js';
+
+const refreshBtn = document.getElementById('refresh');
+const submitBtn = document.getElementById('submit');
+const insertedName = document.getElementById('inserted-name');
+const insertedScore = document.getElementById('inserted-score');
+const fetchedList = document.getElementById('fetched-list');
+
+// let scoreBoard = [];
+
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/';
+
+const displayScore = async () => {
+  fetchedList.innerHTML = '';
+
+  await fetch(api)
+    .then((response) => response.json())
+    .then((json) => {
+      const scoreBoard = json.result;
+      scoreBoard.forEach((item) => {
+        const list = document.createElement('li');
+        list.innerHTML = `
+        <li>${item.user} ${'scored'} ${item.score} ${'points'}</li><br>`;
+        fetchedList.appendChild(list);
+      });
+    });
+};
+
+refreshBtn.addEventListener('click', () => {
+  displayScore();
+});
```

Comment:
- [ ] Your app does not work. When load the app, no scores are displayed (which is okay), but when I click the refresh button nothing happens still. I think you may want to look at your console log errors in the browser.

  ![image](https://github.com/ANTOINE1128/Leaderboard/assets/54848363/a2a72286-12be-4053-a30e-d781732bee15)